modules/mgmt-power-remediation-baremetal-about.adoc

```diff
@@ -0,0 +1,113 @@
+// Module included in the following assemblies:
+
+// * machine_management/mgmt-power-remediation-baremetal
+
+[id="mgmt-power-remediation-baremetal-about_{context}"]
+= About power-based remediation of bare metal
+In a bare metal cluster, remediation of nodes is critical to ensuring the overall health of the cluster. Physically remediating a cluster can be challenging and any delay in putting the machine into a safe or an operational state increases the time the cluster remains in a degraded state, and the risk that subsequent failures might bring the cluster offline. Power-based remediation helps counter such challenges.
+
+Instead of reprovisioning the nodes, power-based remediation uses a power controller to power off an inoperable node. This type of remediation is also called power fencing.
+
+{product-title} uses the `MachineHealthCheck` controller to detect faulty bare metal nodes by initiating power-cycling. Power-based remediation is fast and reboots unhealthy hosts instead of removing them from the cluster.
+
+Power-based remediation provides the following capabilities:
+
+* Allows the recovery of control plane nodes
+* Reduces the risk data loss in hyperconverged environments
+* Reduces the downtime associated with recovering physical machines
+
+[id="machine-health-checks-bare-metal_{context}"]
+== MachineHealthChecks on bare metal
+
+Machine deletion on bare metal cluster triggers reprovisioning of a bare metal host.
+Usually bare metal reprovisioning is a lengthy process, during which the cluster
+is missing compute resources and applications might be interrupted.
+To change the default remediation process from machine deletion to host power-cycle,
+annotate the `MachineHealthCheck` resource with the
+`machine.openshift.io/remediation-strategy: external-baremetal` annotation.
+
+After you set the annotation, unhealthy machines are power-cycled by using
+BMC credentials.
+
+[id="mgmt-understanding-remediation-process_{context}"]
+== Understanding the remediation process
+
+The remediation process operates as follows:
+
+. The MachineHealthCheck (MHC) controller detects that a node is unhealthy.
+. The MHC notifies the bare metal machine controller requests to power-off the unhealthy node.
```

Comment:
Something is slightly off here?
maybe:
The MHC notifies the bare metal machine controller **which** requests to power-off the unhealthy node.